blocks/embedded-form/embedded-form.css

```diff
@@ -1 +1,151 @@
 @import url("../../forms/form.css");
```

Comment:
Import the common form styling elements to the embedded form block to prevent the need from having those styling rules in two places.  This CSS just styles the block up to the form wrapper